docs-2.0/5.configurations-and-logs/1.configurations/1.get-configurations.md

```diff
@@ -0,0 +1,34 @@
+# Get configurations
+
+This document gives two methods to get configurations in Nebula Graph.
```

Comment:
Piece of advice: do not use a specific number to describe the number of items in the following content. Because it is common that we'll have to add a new item, for example, a new method as we update the doc.